### PR TITLE
Refactored Status RPC

### PR DESCRIPTION
**Pull request by dissoupov**

### CODE REVIEW

File: backend/service/status/status_test.go [hunk 1]
```diff
@@ -38,6 +39,16 @@ var (
...
 )
 
+var jsonContentHeaders = map[string]string{
+	header.Accept:      header.ApplicationJSON,
+	header.ContentType: header.ApplicationJSON,
+}
+
+var textContentHeaders = map[string]string{
+	header.Accept:      header.TextPlain,
+	header.ContentType: header.ApplicationJSON,
```
Comment (haykbaluyan) [inline backend/service/status/status_test.go:49]: Should not the contentype header also be header.TextPlain?

File: pkg/print/v1pb.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.Append([]string{"Name", r.Status.Name})
+	table.Append([]string{"Node", r.Status.Nodename})
+	table.Append([]string{"Host", r.Status.Hostname})
+	table.Append([]string{"Listen URLs", strings.Join(r.Status.ListenURLs, ",")})
+	table.Append([]string{"Version", r.Version.Build})
+	table.Append([]string{"Runtime", r.Version.Runtime})
+
+	startedAt := time.Unix(r.Status.StartedAt, 0)
+	uptime := time.Now().Sub(startedAt) / time.Second * time.Second
```
Comment (haykbaluyan) [inline pkg/print/v1pb.go:32]: @dissoupov could you remind why here we divide by time.Second and then multiply by the same time.Second?

File: scripts/build/genproto.sh [hunk 3]
```diff
@@ -41,8 +41,8 @@ function cleanup {
   rm -rf "${GOPATH}"
 }
 
-cleanup
-trap cleanup EXIT
+#cleanup
```
Comment (haykbaluyan) [inline scripts/build/genproto.sh:44]: If not used can we remove? GIT anyways keeps the version and usually, this type of comment ends up cluttering and being forgotten.

**Review by haykbaluyan — APPROVED** [review]

File: api/v1/trustypb/rpc.pb.go [hunk 4]
```diff
@@ -722,15 +783,44 @@ func (m *VersionResponse) Unmarshal(dAtA []byte) error {
...
-			return fmt.Errorf("proto: VersionResponse: illegal tag %d (wire type %d)", fieldNum, wire)
+			return fmt.Errorf("proto: ServerVersion: illegal tag %d (wire type %d)", fieldNum, wire)
 		}
 		switch fieldNum {
 		case 1:
 			if wireType != 2 {
-				return fmt.Errorf("proto: wrong wireType = %d for field Version", wireType)
+				return fmt.Errorf("proto: wrong wireType = %d for field Build", wireType)
+			}
+			var stringLen uint64
```
[the remark this answers is not in the document]
Comment (dissoupov) [reply api/v1/trustypb/rpc.pb.go:796]: this is output from proto compiler for go

File: backend/service/status/status_test.go (hunk 1 above)
Comment (dissoupov) [reply backend/service/status/status_test.go:49]: `Content-Type` what is sent. 
`Accept` was it wants back

File: pkg/print/v1pb.go (hunk 2 above)
Comment (dissoupov) [reply pkg/print/v1pb.go:32]: to round up `Uptime`  to the sec. So output will be:
`1h`, or `1h37m` or `1h37m15s` or `15s`
and ms will be dropped

**Review by haykbaluyan — APPROVED** [review]

File: scripts/build/genproto.sh (hunk 3 above)
Comment (dissoupov) [reply scripts/build/genproto.sh:44]: Have not decided yet. Ideally we need to use `go mod` and add to vendors, but `proto` files are omitted in vendors so have to clone entire repo.
will need to improve this script

**Pull request merged**